src/Pages/OATEditorPage/OATEditorPage.tsx

```diff
@@ -5,12 +5,14 @@ import OATGraphViewer from '../../Components/OATGraphViewer/OATGraphViewer';
 import OATPropertyEditor from '../../Components/OATPropertyEditor/OATPropertyEditor';
 import OATImport from './Internal/OATImport';
 import { getEditorPageStyles } from './OATEditorPage.Styles';
+import { ErrorBoundary } from 'react-error-boundary';
```

Comment:
Is this the package used by the `iot-cardboard-js` team in their work?

Reply:
No, on other `pages` the use of `{ ErrorBoundary } from 'react-error-boundary'` was not found.
Most similar approach found is 
`{ withErrorBoundary } from '.../Models/Context/ErrorBoundary'` where function wraps component  withErrorBoundary(Component) and replaces it with a gray error box whenever an error occurs

Reply:
Is this a new package then? If so, are we missing a package.json change?

Reply:
It just worked on import, I did not have to install it as a new package.
So, I think it is already pre enabled